app/assets/javascripts/login.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```

Comment:
If you use generators and they generate js files that you don't use, delete them.